src/pages/quizPage/QuizPage.js

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect } from "react";
+import "./quizPage.css";
+import axios from "axios";
+import { QuestionCard } from "../../component";
+import { useQuiz } from "../../context/quiz-context";
```

Comment:
You can create index.js for all the context & then import them in single line here. This is fine as well though if you have less contexts

Reply:
There are not many contexts here